main.css

```diff
@@ -161,7 +160,7 @@
 }
 
 .links:hover {
-    color: #fff;
+    color: #000;
```

Comment:
A point mentioned on another PR around colour consistency - we seem to have a mix of words, RGB and hex. Think it would be good to stick to one.

Reply:
Should we change RGB ---> HEX. As hex is used in more areas than RGB in our code so It'll be easy to replace?
What does other's think about this?

Reply:
Done! Changed all RGBA, color text --> HEX